specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -4298,6 +4298,42 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}": {
```

Comment:
you should support collection GET i.e list as well

Reply:
1. What do you mean "support collection GET".? Current API is a already get:
/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}
2. For now, we don't have list use scenarios, but I can add list, the url will be
/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations
Does this list url looks good to you?

Reply:
yes, by collection GET I meant list (sorry if it confused with CosmosDB's concept of collection) - and the url looks good